spec/controllers/admin/application_controller_spec.rb

```diff
@@ -0,0 +1,43 @@
+require "rails_helper"
+
+module Admin
+  class OrdersController < Admin::ApplicationController
+    def after_resource_destroyed_path(_requested_resource)
+      { action: :index, controller: :customers }
+    end
+
+    def after_resource_created_path(requested_resource)
+      [namespace, requested_resource.customer]
+    end
+
+    def after_resource_updated_path(requested_resource)
+      [namespace, requested_resource.customer]
+    end
+  end
+end
+
+RSpec.describe Admin::OrdersController, type: :controller do
```

Comment:
Following my comment above, I think you can implement the same effect here, like this:

```suggestion
RSpec.describe Admin::OrdersController, type: :controller do
  controller do
    def after_resource_destroyed_path(_requested_resource)
      { action: :index, controller: :customers }
    end

    def after_resource_created_path(requested_resource)
      [namespace, requested_resource.customer]
    end

    def after_resource_updated_path(requested_resource)
      [namespace, requested_resource.customer]
    end
  end

```

Reply:
we still have to use controller(Admin::OrdersController), but thanks a lot, good to learn that